### DISCUSSION TITLE
Welcome to confluent-kafka-python Discussions!

### DISCUSSION THREAD

**Discussion by jliunyu** [confluentinc/confluent-kafka-python#1274] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions about confluent-kafka-python here in discussions rather than opening a Github issue.
- Share ideas.
- Engage with other community members, help out, be friendly.

**Comment by qingtian-glo** [comment]
Hi @jliunyu, about the "The SerializingProducer is an experimental API and subject to change." comment in the library, I want to ask if confluent is going to stabilise the SerializingProducer or not? Would it be production ready someday?

Thank you very much.

**Comment by bharti-gwalani** [comment]
Hi, to support high availability we have 2 schema registry instances in our cluster.  But in schema_registry_client.py there is provision to supply just one url.  Is there a work around?
Thanks!
Bharti

**Reply by OneCricketeer** [reply]
@bharti-gwalani The "one url" in the config should accept a CSV list of addresses
